EnglishFilter/sections/whitelist.txt

```diff
@@ -491,6 +491,8 @@ lmgtfy.app#@##bottom_ads
 4snip.pw#@##DividerAd
 ! neads.io is broken
 @@||neads.io^$domain=neads.io
+! https://github.com/AdguardTeam/AdguardFilters/issues/71639
+@@||static.gumtree.com.au^$domain=gumtree.com.au
```

Comment:
Wrong filter. It's caused by Liste FR. Add the exception to the AdGuard French.

Reply:
Btw, is it really necessary to whitelist this domain? Something like that `@@/myads.$domain=gumtree.com.au` is enough.

Reply:
@zloyden, maybe `/myads.$domain=~gumtree.com.au` then?

Reply:
It's better to add the exception here because of `/myads.$domain=~myads.agency`.